provisioning/miniprov/hare_mp/main.py

```diff
@@ -601,6 +601,18 @@ def generate_support_bundle(args):
         if args.t:
             cmd.append('-t')
             cmd.append(args.t[0])
+        if args.duration:
+            print("Duration: ", args.duration)
```

Comment:
Instead of printing, you can raise an exception, `raise NotImplementedError('xxx filter is not implemented')`.

Reply:
Addressed.